plugin/trino-hive/src/test/java/io/trino/plugin/hive/crypto/TestHiveParquetEncryption.java

```diff
@@ -291,8 +291,6 @@ private static void writeTwoColumnEncryptedDictionaryFile(java.nio.file.Path pat
                     .filter(column -> column.getPath().equals(ID_PATH))
                     .findFirst().orElseThrow();
 
-            assertThat(ageChunk.getDictionaryPageOffset()).isGreaterThan(0);
-            assertThat(idChunk.getDictionaryPageOffset()).isGreaterThan(0);
```

Comment:
I'm not sure this will address the problem
If a column uses dictionary encoding, then it must write a dictionary page and that has to be a non-zero offset from beginning of file (because parquet file has a compulsory header of few bytes).
So if this was failing sometimes, then the next check for use of dictionary encoding should also fail, as the dictionary page offset can be zero only if there is no dictionary page at all.